bahmni-lite/.env

```diff
@@ -9,6 +9,8 @@ OPENMRS_HOST=openmrs
 OPENMRS_PORT=8080
 OPENMRS_ATOMFEED_USER=admin
 OPENMRS_ATOMFEED_PASSWORD=Admin123
+OPENMRS_SYSTEM_SMS_USER=systemuser
```

Comment:
Can we change this to `system-user` please. Easier to read.

Reply:
Sure